app/views/root/_filter.html.erb

```diff
@@ -0,0 +1,81 @@
+<%
+  assignees = [{text: "All assignees", value: ""}]
+  content_types = []
+  states = []
+
+  @presenter.available_users.map do | user |
```

Comment:
We should probably be consistent with what we do for the content type filter, and either use a helper method, or just move this code to the `FilteredEditionsPresenter`, since the `available_users` method is only called in this one place and I don't think the presenter being coupled to the view like that would be a problem (it would also make it easy to unit-test).